packages/main/src/RatingIndicator.js

```diff
@@ -207,6 +207,7 @@ class RatingIndicator extends UI5Element {
 				selected: i <= tempValue,
 				index: i,
 				halfStar,
+				readonly: this.readonly,
```

Comment:
This will have the same value for all items. You don't need to push it here in order to access it in the `.hbs` file. Use the `../` syntax